Documentation/helm-values.rst

```diff
@@ -1,6 +1,9 @@
 ..
   AUTO-GENERATED. Please DO NOT edit manually.
 
+.. role:: raw-html-m2r(raw)
+   :format: html
```

Comment:
Curious how this got added, is this automatic with the newer m2r? I didn't notice before. Does it render correctly?

Reply:
I'm not sure; I definitely didn't add this manually. Is there an easy way to preview docs?

Reply:
https://docs.cilium.io/en/latest/contributing/docs/docstest/

Reply:
Perfect. It doesn't seem to change the rendering, as far as I can see.